mne/viz/topo.py

```diff
@@ -152,6 +153,11 @@ def _plot_topo(info, times, show_func, click_func=None, layout=None,
     """Helper function to plot on sensor layout."""
     import matplotlib.pyplot as plt
 
+    if layout.kind == 'custom':
+        layout = copy(layout)
+        for i_dim in layout.pos.T[:2]:
+            i_dim -= i_dim.min(0)
+            i_dim /= (i_dim.max(0) - i_dim.min(0))
```

Comment:
Is this used for plotting things on a head outline? If so, then it's misleading, won't work well for ECoG, and won't work well for any channel type after `pick_channels` operations. If it's just for selection, without plotting on a head, it's fine.